corehq/motech/dhis2/tests/test_events_helpers.py

```diff
@@ -84,7 +84,7 @@ def test_form_processing(self):
                 },
                 'org_unit_id': {
                     'doc_type': 'FormUserAncestorLocationField',
-                    'location_field': LOCATION_DHIS_ID
+                    'form_user_ancestor_location_field': LOCATION_DHIS_ID
```

Comment:
That is not a bad idea! `Schema` can transform data during validation. The [Wrap validated data](https://github.com/dimagi/commcare-hq/pull/25961/commits/7c379ab09a1311a0f13939332fb48a77689cea95) commit uses `Schema.validate()` instead of `is_valid()` to allow us to do that. So I think we should!